### DISCUSSION TITLE
Where can I access info about 404 problems (MODIFIEE)

### DISCUSSION THREAD

**Discussion by StrangeRanger** [YOURLS/YOURLS#3372] [Q&A] [answered]
I pretty much have the same problem as described in the [following discussion](https://github.com/orgs/YOURLS/discussions/3175). It looks like the information about 404 stuff might have been moved to the website, since there is no longer a wiki, but I'm not sure where I can find that information. Could I get a link to it.

**Comment by LeoColomb** [comment, marked as answer]
The links on that answers has been updated.
